bundles/org.openhab.binding.wolfsmartset/src/main/java/org/openhab/binding/wolfsmartset/internal/handler/WolfSmartsetUnitThingHandler.java

```diff
@@ -185,7 +185,7 @@ public void updateConfiguration(SubMenuEntryDTO submenu, MenuItemTabViewDTO tabm
     public void updateValues(@Nullable GetParameterValuesDTO values) {
         var thingId = thing.getUID();
         if (values != null && values.getValues() != null && values.getValues().size() > 0) {
-            if (!values.getIsNewJobCreated()) {
+            if (Boolean.FALSE.equals(values.getIsNewJobCreated())) {
```

Comment:
Now that the method returns a boolean, you have to test like this:

if `(!values.getIsNewJobCreated())`

I am not sure if the correct fix would not have been to check null here rather than in the called method. I let you think about that and decide.